### PR TITLE
Bugfix/android player not rendering elements

### PR DESCRIPTION
**Pull request by rubenpingol-xibo**
relates to xibosignageltd/xibo-private#810

### CODE REVIEW

File: modules/layout-renderer.twig [hunk 1]
```diff
@@ -31,14 +31,10 @@
         <meta name="public-path" content="{{ theme.rootUri() }}"/>
         
         <link rel="shortcut icon" href="{{ theme.uri("img/favicon.ico") }}" />
-        <link rel="stylesheet" type="text/css" href="{{ theme.rootUri() }}dist/preview/html-preview.css" />
-        <link rel="stylesheet" type="text/css" href="{{ theme.uri("css/html-preview.css") }}" />
```
Comment (dasgarner) [inline modules/layout-renderer.twig:35]: This file is used during the white label process to replace the Xibo logo with the white label logo equivalent, which I guess is something we need to handle in XLR?

**Review by dasgarner — COMMENTED** [review]
I have a question which might result in a follow on issue. For now, this is good to test and see if it solves the main issue.

**Review by dasgarner — DISMISSED** [review]

File: modules/layout-renderer.twig (hunk 1 above)
Comment (rubenpingol-xibo) [reply modules/layout-renderer.twig:35]: Yes, I guess we can have this handled by XLR instead. I just need to know what data determines if it's white label or not.

**Review by dasgarner — APPROVED** [review]

**Comment by ifarzana** [comment]
TEST: /Layout END

**Pull request merged**